src/utils/web3.js

```diff
@@ -4,7 +4,7 @@ const networkInfo = {
   43112: 'Localhost',
 }
 
-const CHAIN_ID = process.env.ETHEREUM_RPC_CHAIN_ID
+const CHAIN_ID = parseInt(process.env.ETHEREUM_RPC_CHAIN_ID) || 43114
```

Comment:
default to mainnet, eh?

Reply:
it's what we're using for hardhat, which i figured most likely if this value is unconfigured?